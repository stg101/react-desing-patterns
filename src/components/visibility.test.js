import React from "react";
import { render } from "@testing-library/react";

import Visibility from "./visibility";

test("Visibility component", () => {
  const { asFragment } = render(
    <Visibility title="Use hook component:" visibility />
  );

  expect(asFragment()).toMatchSnapshot();
});
