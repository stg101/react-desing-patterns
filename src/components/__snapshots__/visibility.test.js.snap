// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Visibility component 1`] = `
<DocumentFragment>
  <li>
    Use hook component: 
    <i>
      true
    </i>
  </li>
</DocumentFragment>
`;
